web/themes/custom/bglobal/bglobal.theme

```diff
@@ -4,3 +4,20 @@
  * @file
  * Functions to support theming.
  */
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function bglobal_theme_suggestions_page_alter(array &$suggestions, array $variables) {
+  /** @var \Drupal\node\NodeInterface $node */
+  if ($node = \Drupal::routeMatch()->getParameter('node')) {
+    $content_type = $node->bundle();
+    // Get the taxonomy term.
+    if ($node->hasField('field_term_ref_school_microsite') && $node->get('field_term_ref_school_microsite')->getValue()) {
+      $term = $node->get('field_term_ref_school_microsite')->first()->entity;
+      $suggestions[] = 'page__' . 'school_' . $term->id();
```

Comment:
Using term id is probably a bad idea because that is volatile and therefore cannot be relied on.

There will be a `field_url_pattern_shortname` on the School taxonomy which we could use instead. It hasn't been created yet and perhaps needs to be renamed as `field_term_shortname`

This would be a plain-text field that has a 20 character limit. 

We should also be sure to cast `strtolower()` on that value before passing it to the template suggestion.

Reply:
I changed this to be `field_microsite_school_shortname` in line with the other fields defined for that vocabulary.